services/blockstorage/test/get_block_headers.go

```diff
@@ -0,0 +1,105 @@
+package test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/orbs-network/orbs-network-go/test"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+)
+
+var _ = Describe("Block storage", func () {
+	When("asked to provide transactions block header", func () {
```

Comment:
* add a test for timeout if block is not persisted
* add a test for block not in the near future
both of these will force you to extract configuration (as you don't want, for instance, to make the test wait 10 seconds)